pkg/neotest/chain/chain.go

```diff
@@ -258,18 +258,14 @@ func NewMultiWithOptionsNoCheck(t testing.TB, options *Options) (*core.Blockchai
 		options = &Options{}
 	}
 
-	cfg := config.Blockchain{
-		ProtocolConfiguration: config.ProtocolConfiguration{
-			Magic:              netmode.UnitTestNet,
-			MaxTraceableBlocks: MaxTraceableBlocks,
-			TimePerBlock:       TimePerBlock,
-			StandbyCommittee:   standByCommittee,
-			ValidatorsCount:    4,
-			VerifyTransactions: true,
-		},
+	cfg, err := config.Load(config.DefaultConfigPath, netmode.UnitTestNet)
+	if err != nil {
+		return nil, nil, nil, err
 	}
+	cfg.ProtocolConfiguration.Hardforks = nil
```

Comment:
It's the same cludge as https://github.com/nspcc-dev/neo-go/pull/3696#discussion_r1872076389, but in the code level. We can't do this. Set 4 hardforks explicitly in the unittest config.